src/webserial.ts

```diff
@@ -1,3 +1,11 @@
+export interface SerialOptions {
+  baudRate?: number | undefined
```

Comment:
Add a comment - baudrate as per the documentation of webserial api is a 'required' field as part of serialoptions. But since we are maintaining baudrate param separately outside and that is actually used in the code, keeping this optional in the dictionary for now.